source/manual/howto-restore-an-individual-postgres-database.html

```diff
@@ -0,0 +1,55 @@
+---
+owner_slack: "#re-govuk"
+title: Restore an Individual Postgres Database in AWS RDS
+section: Backups
+layout: manual_layout
+parent: "/manual.html"
+last_reviewed_on: 2019-12-10
+review_in: 3 months
+---
+
+> If you need to restore an entire RDS instance please see these [instructions](https://docs.publishing.service.gov.uk/manual/howto-backup-and-restore-in-aws-rds.html) instead.
+
+### Restoring From S3 Backups
+
+We take nightly database backups and save them to the `govuk-production-database-backups` S3 bucket inside the `postgresql-backend` folder.
+
+To restore a backup from the most recent copy follow the steps below:
+1. SSH into the db_admin machine. If you are using GDS CLI you can run the following command to ssh into the correct machine:
+```
+gds govuk connect ssh -e <environment> db_admin```
+
+2. To find the command you need to run to restore a specific database, list the the govuk-backup user's cronjobs, then grep for the database name you need:
```

Comment:
Markdown trick: if you set all of these numbers to `1.`, they will automatically convert to the step numbers when the page gets rendered. Saves you the effort of re-numbering when you add/remove a step.

Reply:
This doesn't appear to work with the code blocks in use as well. Huh.